sql/core/src/main/scala/org/apache/spark/sql/execution/columnar/ColumnStats.scala

```diff
@@ -295,6 +295,28 @@ private[columnar] final class BinaryColumnStats extends ColumnStats {
     Array[Any](null, null, nullCount, count, sizeInBytes)
 }
 
+private[columnar] final class IntervalColumnStats extends ColumnStats {
+  protected var upper: CalendarInterval =
+    new CalendarInterval(Int.MinValue, Int.MinValue, Long.MinValue)
```

Comment:
How about defining constant values for min/max, e.g., `CalendarInterval.MinValue`?

Reply:
this is nice